tests/functions/test_groupby_agg.py

```diff
@@ -59,3 +59,32 @@ def test_groupby_agg_multi():
     expected_agg = np.array([1, 1, 1, 1, 1, 1])
 
     np.testing.assert_equal(df_new["date_average"], expected_agg)
+
+
+@pytest.mark.functions
+def test_groupby_agg_multi_column():
```

Comment:
@jiafengkevinchen: I was a bit confused by the test name when I first read the test - would you be kind enough to add a docstring description here to clarify? 

Here's a suggested starter:

```suggestion
def test_groupby_agg_multi_column():
    """Test for the case when we want to groupby one column and agg on another, while leaving out other columns."""
```

The docstring purpose is so that I (or other maintainers) can remember what the test intent is.

(Don't forget to make the linter checks pass too!)